src/low-level/smtp/mailsmtp.c

```diff
@@ -448,6 +448,33 @@ int mailsmtp_data_message(mailsmtp * session,
   }
 }
 
+/* lmtp data command
+ * on end of data transaction lmtp return a status for
+ * every recipient it delivered. the status is stored in
+ * a carray retcodes. the array must have the size of recips */
```

Comment:
Do you need a status for each recipient or could it return one single failure for everything?